src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -0,0 +1,133 @@
+---
+title: API Changes from 1.5 to 2.0
+layout: default
+---
+
+# UI Options 2.0 Migration #
```

Comment:
The level one heading should probably be "API Changes from 1.5 to 2.0" – UIO migration things could be grouped under a level two heading?